Sources/SwiftProtobufCore/JSONEncoder.swift

```diff
@@ -102,18 +102,14 @@ internal struct JSONEncoder {
         data.append(contentsOf: text.utf8)
     }
 
-    internal mutating func append(bytes: [UInt8]) {
-        data.append(contentsOf: bytes)
-    }
-
     /// Append a raw utf8 in a `Data` to the JSON text.
     internal mutating func append(utf8Data: Data) {
         data.append(contentsOf: utf8Data)
     }
 
     /// Append a raw utf8 in a `[UInt8]` to the JSON text.
-    internal mutating func append(utf8Data: [UInt8]) {
-        data.append(contentsOf: utf8Data)
+    internal mutating func append(bytes: [UInt8]) {
```

Comment:
Drive by while you are working on things -- should this keep `utf8` in the name to make it clear it isn't arbitrary bytes, it has to be a UTF8 sequence that is being appended?